rom/laghos_rom.cpp

```diff
@@ -2277,6 +2288,28 @@ void ROM_Basis::readSP(ROM_Options const& input, const int window)
     }
 }
 
+void ROM_Basis::writePD(const int id, const int window) const
```

Comment:
Would it be clearer to call this function `writePDweights`?

Reply:
I agree. I have changed the name of the function.